src/Backend/Core/Layout/Templates/FormLayout.html.twig

```diff
@@ -303,7 +303,7 @@
                   {% endif %}
                   {% if form.vars.allow_delete %}
                     <div class="btn-group pull-right">
-                      <a href="#" class="btn btn-danger btn-sm" data-removefield="collection" data-field="{{ field.vars.id }}">{{ form.vars.delete_button_text|trans({}, translation_domain)|raw }}</a>
+                      <a href="#" class="btn btn-danger btn-sm" data-removefield="collection" data-field="{{ field.vars.id }}">{{ form.vars.delete_button_text|trans({}, translation_domain)|ucfirst|raw }}</a>
```

Comment:
Maybe switch this over to buttons and test if the js still works in that case

Reply:
I have switched the anchor tags to buttons and tested the js on the latest Fork release
![screen shot 2019-01-30 at 11 58 51](https://user-images.githubusercontent.com/40020038/51977014-92493680-2486-11e9-848d-b727ba059c38.png)